docs/source/prerequisites/index.rst

```diff
@@ -8,12 +8,10 @@ Prerequisites
 * **initiator**: :python:`Python <>` and :pipx:`pipx <>`
 * **target**: ssh-server
 
-For the patient reader, the rest of this section provides details on meeting
-these requirements, along with various system setup instructions. The system
-setup is divided into **initiator** and **target** requirements.
-
-To refresh your understanding from the :ref:`sec-introduction` section, these
-roles are illustrated as follows:
+For readers who prefer more details, the rest of this section provides
+additional information and is separated into **initiator** and **target**
+requirements. Starting with a "refresher" in the illustration below of what the
+**initiator** and **target** systems cover.
```

Comment:
Maybe just "The diagram below illustrates what the **initiator** and **target** systems cover." to keep the language a bit more formal?